src/test/java/Utility/MockObserver.java

```diff
@@ -0,0 +1,10 @@
+package Utility;
+
+import Contract.Contract;
+
+public class MockObserver implements Contract.Observer {
+    @Override
+    public void update() {
+
+    }
+}
```

Comment:
this works, but cleaner would be to use a mocking library, like [Mockito](https://site.mockito.org/)

Reply:
Yeah I knew that, but just wanted to get this done quickly and didn't want to spend time working out mockito